js/src/jupyter-leaflet.js

```diff
@@ -583,10 +578,16 @@ var LeafletMapView = widgets.DOMWidgetView.extend({
         }
     },
 
-    onResize: function (msg) {
-        this.obj.invalidateSize(true);
-    }
-
+    processPhosphorMessage: function(msg) {
+        LeafletMapView.__super__.processPhosphorMessage.apply(this, arguments);
+            switch (msg.type) {
+                case 'resize':
+                    this.obj.invalidateSize(true);
```

Comment:
missing break statements, this will get called twice

Reply:
thanks that was it. under my nose!

Reply:
Fixed.